Need to add one step in quick start procedure of About Logging 6.0 documentation (#86648)
openshift/openshift-docs: observability/logging/logging-6.0/log6x-about.adoc

. Install the `OpenShift Logging` and `Loki` Operators from OperatorHub.

> . Create a secret with the details to access to an existing object storage bucket. AWS is used in the following example: 
> +

Reviewer: ```suggestion
. Create a secret to access an existing object storage bucket:  
+
.Example command for AWS
```

Author: Hello @xenolinux ,
Thank you very much for reviewing this PR and providing some suggestion.

Sure.
Let me try to commit the suggested changes and squash the changes.

Regards,
Prithviraj Patil
